ingress/ingress.yml

```diff
@@ -0,0 +1,16 @@
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: todoapp-ingress
+  namespace: todoapp
+spec:
+  rules:
+    - http:
+        paths:
+          - path: /
```

Comment:
![image](https://github.com/mate-academy/devops_todolist_kubernetes_task_10_ingress/assets/87720583/34632a9f-2c64-4541-aac4-cd3cc596d055)
Need to add regex